finagle-example/README

```diff
@@ -1,12 +1,12 @@
 # How to run the examples:
 
-From the command line:
+From the command line in cloned directory `finangle`:
```

Comment:
git doesn't require that the cloned directory be named finagle, maybe we should say the root directory of the finagle repository?